portal/templates/dashboard/deals.njk

```diff
@@ -31,151 +31,140 @@
     }]
   }) }}
 
-  <div class="govuk-tabs" data-module="govuk-tabs">
-    <h2 class="govuk-tabs__title">Contents</h2>
-
-    {{ dashboardTabs.nav(selected = tab) }}
-
-    <section class="govuk-tabs__panel">
-
-       <div class="govuk-grid-row">
-
-         <div class="govuk-grid-column-three-quarters">
-            &nbsp
-            {# TODO - Enable aside filters #}
-            {# {{ mojButtonMenu({
-               items: [{
-                  text: "Show filter",
-                  href: "#",
-                  classes: "govuk-button--secondary"
-                  }]
-            }) }} #}
-         </div>
-
-         <div class=" govuk-grid-column-one-quarter">
-            <div class="govuk-!-padding-left-2">
-               <form method="POST" autocomplete="off">
-                  {{ govukCheckboxes({
-                     name: "createdByYou",
-                     classes: "govuk-checkboxes--small",
-                     items: [
-                           {
-                              value: "true",
-                              text: "Created by you",
-                              checked: createdByYou,
-                              attributes: {
-                                 "data-cy": "created-by-you",
-                                 'onChange': 'this.form.submit()'
-                              }
-                           }
-                     ]
-                  }) }}
-                  <div class="moj-button-action">
-                     {{ govukButton({
-                           text: 'Submit',
-                           classes: 'moj-js-hidden',
-                           attributes: {
-                              'data-cy': 'submit-created-by-you'
-                           }
-                     }) }}
-                  </div>
-
-               </form>
-            </div>
-         </div>
+  {{ dashboardSubNav.subnav(selected = tab) }}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-three-quarters">
+      &nbsp
+      {# TODO - Enable aside filters #}
```

Comment:
should we just remove this so we don't have TODOs littered in the code? Appreciate it wasn't added in this branch